.nf-core.yml

```
@@ -9,4 +9,4 @@ template:
   name: lsmquant
   org: nf-core
   outdir: .
```

```suggestion
  outdir: .
  skip_features:
  	- fastqc
  	- igenomes
```
this will make future template syncs easier, otherwise you will also need to manually remove related sections from the different files